### DISCUSSION TITLE
"Quit App after media sync" - discovery

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#207] [General]
Hi,
I experienced a little issue in trying to reverse the option to "Quit app after media sync". The JWMMF app would shut down before I could see / modify settings. Perhaps I could recommend a minor website adjustment. 
Not a biggie - just may assist others that's all. 

[<img src="https://simonsheena-gmail.tinytake.com/media/1228731?filename=1645317359570_Quit+App.png&sub_type=thumbnail_preview&type=attachment&width=798&height=351" title="Powered by TinyTake Screen Capture"/><br><a href="https://www.tinytake.com">Powered by TinyTake Screen Capture</a>](url)

**Comment by sircharlo** [comment]
Thanks for that! I've updated the docs with that suggestion. ([Here](https://github.com/sircharlo/jw-meeting-media-fetcher/commit/fb9af27478542470376675cfce2f47935cb4e4aa#diff-286fd3f2e5a94297adb6cce6f7acd535c25978530c99fd2999df96d941f60470R18) and [here](https://github.com/sircharlo/jw-meeting-media-fetcher/commit/fb9af27478542470376675cfce2f47935cb4e4aa#diff-286fd3f2e5a94297adb6cce6f7acd535c25978530c99fd2999df96d941f60470R20).)
